streams/src/test/java/org/apache/kafka/streams/KafkaStreamsTest.java

```diff
@@ -530,4 +535,42 @@ public void onChange(final KafkaStreams.State newState, final KafkaStreams.State
             mapStates.put(newState, prevCount + 1);
         }
     }
+
+    @Test
+    public void testLogContext() {
+        final UnitTestAppender appender = new UnitTestAppender();
+        final Logger logger = Logger.getRootLogger();
+        logger.addAppender(appender);
+
+        try {
+            Logger.getLogger(KafkaStreams.class);
+            streams.start();
+        } finally {
+            logger.removeAppender(appender);
+            streams.close();
+        }
+
+        final LoggingEvent firstLogEntry = appender.log.get(0);
+        Assert.assertEquals("org.apache.kafka.streams.KafkaStreams", firstLogEntry.getLoggerName());
+        Assert.assertTrue(firstLogEntry.getMessage().toString().contains("[stream-client="));
+    }
+
+    class UnitTestAppender extends AppenderSkeleton {
```

Comment:
Could we move it to a separate class and rename to `MockAppender`, assuming it would be used for a lot more class tests that @dguy pointed out that need to be modified as well?

Reply:
EDIT: Just saw @dguy 's comment, and I agree that we do not need to this test.

Reply:
Sure, will remove the related unit test above. Thanks!